src/meta/src/stream/stream_graph/visit.rs

```diff
@@ -93,6 +93,9 @@ where
                         always!(s.table, "HashAgg");
                     }
                 }
+                for (distinct_col, dedup_table) in &mut node.distinct_dedup_tables {
```

Comment:
How could it work without assigning a global table ID before this PR? 🥵

Reply:
I think table id is assigned in this PR😄

Reply:
If the table ID is local for each fragment graph (mview), then tables from multiple views may get conflicted. 😄 I guess it'll be better to assign a range, for example, from 1000, for global table IDs, then we can check it out earlier. Not a big deal.